config/cucumber.yml

```diff
@@ -18,7 +18,7 @@
   end
 %>
 
-default: --profile devel
+default: --publish-quiet --profile devel
```

Comment:
what is this for?

Reply:
already done with https://github.com/openshift/verification-tests/pull/1989...so redundant.

Reply:
If this is this [1], then I don't think it is a good idea - to publish our logs to a public service.

[1]  https://cucumber.io/blog/open-source/cucumber-reports/

Reply:
@akostadinov #1989 already disabled printing that marketing stuff.  That line just disables cucumber from printing that marketing text at the end of the runs.